### DISCUSSION TITLE
(Feature request) Show missing host information

### DISCUSSION THREAD

**Discussion by ghost** [aldostools/webMAN-MOD#973] [Ideas]
I.e it should show Connected From:Your IP accessed through webMAN MOD on either DEX or PEX. Close this if you agree for it to be implemented in Cobra only maybe.

**Comment by aldostools** [comment]
It would saturate the screen with popup messages when accessed remotely (e.g. from PC, tablet or phone)

I prefer not implement this feature. On DEX you can show the remote IP connected when you use ProDG Target Manager.
